quinn/src/lib.rs

```diff
@@ -115,3 +115,8 @@ const RECV_TIME_BOUND: Duration = Duration::from_micros(50);
 
 /// The maximum amount of time that should be spent in `sendmsg()` calls per endpoint iteration
 const SEND_TIME_BOUND: Duration = Duration::from_micros(50);
+
+/// The maximum size of content length of packets in the outgoing transmit queue. Transmit packets
+/// generated from the endpoint (retry or initial close) can be dropped when this limit is being execeeded.
+/// Chose to represent 100 MB of data.
+const MAX_TRANSMIT_QUEUE_CONTENTS_LEN: usize = 10_0000_000;
```

Comment:
I think you meant to write `100_000_000`?

Reply:
oh. yes

Reply:
You left the extra 0 in there, making this 1G, which I don't think was intended.

Reply:
The 100MB was intended not 10MB. With 100MB. Running a simple QUIC server which is under the persistent flood of initial packets actually shows RES memory usage of more than 2g.

Reply:
The value currently written is 1G, not 100MB.

Reply:
Oh. My bad! Sorry. Updating it to true 100MB